bddl/activity_definitions/buy_a_microwave_oven/problem0.bddl

```diff
@@ -1,33 +0,0 @@
-(define (problem buy_a_microwave_oven-0)
```

Comment:
It seemed that you are completely removing some of these activities. Is it because they are not part of B1K?

Reply:
yes